#159236205 Upgrade the nutrition diary

Yiga-fred opened the pull request:
#### What does this PR do?
Enable a user to add a meal already eaten.
#### Description of Task to be completed?
The user should be able to enter the food he actually ate.
#### How should this be manually tested?
After fetching the branch **ft-Add-already-eaten-meal-159236205**
- Run the following commands.
     - `python manage.py makemigrations`
     - `python manage.py migrate`
- Run the application `python manage.py runserver`
- Login with any registered user and navigate to the nutrition plans page.
#### Any background context you want to provide?
Before this implementation, a user could only add a meal he/she is planning to eat.
#### Screenshots
<img width="934" alt="screen shot 2018-08-23 at 07 52 49" src="https://user-images.githubusercontent.com/39955261/44505362-b0299880-a6a9-11e8-96a8-a4651c51ccf2.png">
<img width="627" alt="screen shot 2018-08-23 at 07 53 07" src="https://user-images.githubusercontent.com/39955261/44505370-b9b30080-a6a9-11e8-8ecc-02190fd19a25.png">
<img width="783" alt="screen shot 2018-08-23 at 07 53 25" src="https://user-images.githubusercontent.com/39955261/44505374-c172a500-a6a9-11e8-8d27-aa2b3361bf4c.png">

wasswa-derick commented on wger/nutrition/forms.py:
```diff
@@ -146,3 +147,14 @@ def __init__(self, *args, **kwargs):
+        exclude = ()
+
+    def __init__(self, *args, **kwargs):
+        super(MealForm, self).__init__(*args, **kwargs)
```
Kindly use python3-style call on `super`

Commit:
[Feature #159236205]Add already eaten meal

Currently a user can only enter into the plan the food they plan to eat

This commit enables the user to enter the food he/she actually ate during the day.

lym commented on wger/nutrition/forms.py:
```diff
@@ -24,7 +24,7 @@
     IngredientWeightUnit,
     Ingredient,
     MealItem,
-    Meal
```
There's no need for this change.

lym commented on wger/nutrition/forms.py:
```diff
@@ -157,4 +157,4 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
-        super(MealForm, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
```
<3
